docs/components/pagination.md

```diff
@@ -11,275 +11,298 @@
   <div>
     <span><b>Visible letters: </b></span>
-    <span v-for="number in data.visibleLetters">{{ number }} </span>
-    <KPagination :items="data.letters" :totalCount="data.letters.length" :pageSizes="[3]" :disablePageJump="true" @pageChanged="({visibleItems}) => data.visibleLetters = visibleItems"/>
+    <span v-for="letter in data.visibleLetters">{{ letter }} </span>
+    <KPagination :items="data.letters" :total-count="data.letters.length" :page-sizes="[3]" disable-page-jump @page-change="({visibleItems}) => data.visibleLetters = visibleItems" />
   </div>
 </KComponent>
 
-```html
+```vue
 <template>
-  <div>
-    <span><b>Visible letters: </b></span>
-    <span v-for="number in data.visibleLetters">{{ number }} </span>
-    <KPagination
-      :items="data.letters"
-      :totalCount="data.letters.length"
-      :pageSizes="[3]"
-      :disablePageJump="true"
-      @pageChanged="({visibleItems}) => data.visibleLetters = visibleItems"/>
-  </div>
+  <span><b>Visible letters: </b></span>
+  <span v-for="letter in visibleLetters">{{ letter }}</span>
+  <KPagination
+    disable-page-jump
+    :items="letters"
+    :total-count="letters.length"
+    :page-sizes="[3]"
+    @page-change="({visibleItems}) => visibleLetters = visibleItems" />
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue'
-
-export default defineComponent({
-  setup() {
-    const letters = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i']
-    const visibleLetters = ['a', 'b', 'c']
-
-    return {
-      letters,
-      visibleLetters,
-    }
-  }
-})
+<script setup lang="ts">
+const letters = ref<string>(['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i'])
+const visibleLetters = ref<string>(['a', 'b', 'c'])
 </script>
 ```
 
 ### currentPage
 
-Manually control the current page instead of using native handling. If using this prop you MUST keep it up-to-date using the `@pageChanged` event in order to remain reactive to clicking the prev, next, and specific page buttons.
+Manually control the current page instead of using native handling. If using this prop you **must** keep it up-to-date using the `@page-change` event in order to remain reactive to clicking the prev, next, and specific page buttons.
 
 <KComponent :data="{ letters: ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i'], visibleLetters: ['d', 'e', 'f'], currPage: 2}" v-slot="{ data }">
   <div>
     <span><b>Visible letters: </b></span>
-    <span v-for="number in data.visibleLetters">{{ number }} </span>
+    <span v-for="letter in data.visibleLetters">{{ letter }}</span>
     <KPagination
       :items="data.letters"
-      :totalCount="data.letters.length"
-      :pageSizes="[3]"
-      :currentPage="data.currPage"
-      @pageChanged="({visibleItems, page}) => { data.visibleLetters = visibleItems; data.currPage = page }"/>
+      :total-count="data.letters.length"
+      :page-sizes="[3]"
```

Comment:
question: why would we only show 1 page size? If this was the case, you wouldn't show the UI element

Reply:
Are you suggesting we don't display page size selection dropdown if there's only 1 option to choose from? This makes sense to me but I feel like we still need to indicate users ho many items per page are displayed...

Reply:
I suggest we display disabled dropdown and hide the caret icon, like so:

![Screenshot 2024-02-29 at 11 30 49 AM](https://github.com/Kong/kongponents/assets/36751160/77d7a5b0-489b-4867-adef-bbcc380cdd6b)

Reply:
I think disabling is fine but curious if it makes sense as an example on the docs site

Reply:
I think it does - those examples are there to serve as example on page navigation, page size has it's own example